### DISCUSSION TITLE
Disable devices like in Sway with disabled_on_external_mouse

### DISCUSSION THREAD

**Discussion by TheFunctionalGuy** [hyprwm/Hyprland#2489] [General]
Hey, 
currently I'm missing a feature to dynamically disable devices based on other connected devices.

Sway for example provides the `disabled_on_external_mouse` option for inputs.
I would like to implement something similar in this fashion:
- You will be able to add a new per device config with the sub member `disables` which can contain a single device name or a list of device names.
- These devices will be disabled when the per-device-config-device gets connected
- These devices will be enabled again when the per-device-config-device gets removed

Example config:
```
device:external-usb-device {
   disables: first-device,other-device
}
```
Where `first-device` and `other-device` would be the devices that get disabled respectively.

Is there an easier way to do that or isn't this feature needed at all?

P.S.: Wrote this from my mobile phone. Please have mercy.

**Comment by MaulanaMalikS** [comment]
Hey,
I'm missing that feature too.
Maybe an issue should be created for this?

**Reply by TheFunctionalGuy** [reply]
If you want to you can do this. Link it afterwards so I can easily reply to it and open a pull request if enough people are interested.

**Reply by MaulanaMalikS** [reply]
I've already created an issue for this #2763.
